crmd/crmd_alerts.c

```diff
@@ -75,6 +75,18 @@ get_meta_attrs_from_cib(xmlNode *basenode, crm_alert_entry_t *entry,
         crm_trace("Found timestamp format string '%s'", value);
     }
 
+    value = g_hash_table_lookup(config_hash, XML_ALERT_ATTR_SELECT_KIND);
+    if (value) {
+        entry->select_kind = (char *) value;
+        crm_trace("Found select_kind string '%s'", entry->select_kind);
+    }
+
+    value = g_hash_table_lookup(config_hash, XML_ALERT_ATTR_SELECT_ATTRIBUTE_NAME);
+    if (value) {
+        entry->select_attribute_name = (char *) value;
+        crm_trace("Found attribute_name string '%s'", entry->select_attribute_name);
+    }
```

Comment:
Okay!
I delete it.

I intend to send the PR that revised your comment in a beginning of the next week.